tests/fixtures/netex_france/arrets.xml

```diff
@@ -3,159 +3,399 @@
 	<PublicationTimestamp>2019-04-03T17:19:00</PublicationTimestamp>
 	<ParticipantRef>Participant</ParticipantRef>
 	<dataObjects>
-		<GeneralFrame>
+		<GeneralFrame id="FR:GeneralFrame:NETEX_ARRET:ProviderCode" version="any">
```

Comment:
<CompositeFrame id="FR:CompositeFrame:NETEX_ARRET:ProviderCode" version="any">
  <frames>
    <ServiceFrame id="FR:GeneralFrame:NETEX_ARRET:ProviderCode" version="any">
@prhod 
CompositeFrame or no to be iso with the lignes.xml file (and netex_idf) ?
I know that the 2 options are valid netex 1.9. It tickles me a little without :smile: